back/src/main/java/com/baba/back/relation/domain/Relation.java

```diff
@@ -52,7 +52,11 @@ public Relation(Member member, Baby baby, String relationName, RelationGroup rel
         this.defaultRelation = defaultRelation;
     }
 
-    public String getBabyId() {
+    public String searchBabyId() {
         return baby.getId();
     }
+
+    public boolean isFamily() {
+        return relationGroup == RelationGroup.FAMILY;
```

Comment:
```suggestion
        return relationGroup.isFamily();
```
이런 식으로 하는 게 더 좋지않을까요?
relationGroup이 RelationGroup.FAMILY인지 Relation 객체가 직접 비교하는 것보다는 relationGroup이 주체적으로 판단할 수 있도록 하는게 더 나을 것 같아요

Reply:
`객체에게 일을 시키자` 원칙이 지키도록 노력해야겠네요! 수정하겠습니다.

Reply:
새로만든 메소드에 대한 테스트 추가가 필요하겠네요

Reply:
좋습니다!